config/alienwhitelist.txt

```diff
@@ -208,3 +208,4 @@ yeehawguvnah - Xenochimera
 zalvine - Shadekin Empathy
 zammyman215 - Vox
 zeracyfr - Xenochimera
+rainbyplays - Protean
```

Comment:
This seems to be the wrong ckey? The linked application is for a tombomb90 but the PR is for rainbyplays?

Both have been approved for Protean so just a matter of correcting the link. Also this should probably be in alphabetical order like the other entries.

Reply:
Oh dear, how embarrassing! My mistake, thanks for the correction. I must have copied the wrong one.